third_party/ngraph/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
+@gunan, @tfboyd for license.

Reply:
Thanks @tfboyd !

We further reviewed the licenses of the dependency libraries  (ngraph, nlohmann_json_lib, ngraph_tf ). They are notice-only licenses and already approved.

Reply:
Thanks @thirupalanisamy and @tfboyd !